janus.c

```diff
@@ -108,18 +108,9 @@ static struct janus_json_parameter admin_parameters[] = {
 	{"transaction", JSON_STRING, JANUS_JSON_PARAM_REQUIRED},
 	{"janus", JSON_STRING, JANUS_JSON_PARAM_REQUIRED}
 };
-static struct janus_json_parameter debug_parameters[] = {
```

Comment:
I understand a helper to set a boolean, but why remove the templates? They're helpful for validation, and we can't always assume that parsing and setting will always be combined. I'd keep the two logically separated.